### DISCUSSION TITLE
Logo Ideas?

### DISCUSSION THREAD

**Discussion by GCuser99** [GCuser99/SeleniumVBA#65] [Ideas]
In researching Inno Setup for the twinBasic Dll, I was confronted with a choice of providing an install logo. Made me think why don't we have one?

Here's a proposal:

![logo](https://user-images.githubusercontent.com/26237126/213843777-09f97d82-4300-4915-9842-f206233e34a6.png)

Anyone  else?

**Comment by 6DiegoDiego9** [comment]
Nice :)

**Comment by GCuser99** [comment]
To celebrate our 20th star milestone :-), I placed it in our ReadMe file as a test - if/when you find it annoying, let me know and I will take it down.
